apps/web/screens/Layouts/SidebarLayout.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, ReactNode } from 'react'
+import {
+  Box,
+  GridContainer,
+  GridRow,
+  GridColumn,
+  Hidden,
+} from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+
+import * as styles from './SidebarLayout.treat'
+import cn from 'classnames'
+type Breakpoint = keyof typeof theme['breakpoints']
+
+interface SidebarLayoutProps {
+  sidebarContent: ReactNode
+  isSticky?: boolean
+  hiddenOnTablet?: boolean
+}
+
+export const SidebarLayout: FC<SidebarLayoutProps> = ({
+  sidebarContent,
+  isSticky = true,
+  hiddenOnTablet = false,
+  children,
+}) => (
+  <GridContainer position="none">
+    <Box
+      display="flex"
+      flexDirection="row"
+      height="full"
+      paddingBottom={6}
+      position={isSticky ? 'relative' : undefined}
+    >
+      <Box
+        printHidden
+        className={cn(styles.sidebarWrapper, isSticky ? styles.sticky : null)}
```

Comment:
classnames has a built in solution for conditional styles using object
```suggestion
        className={cn(styles.sidebarWrapper, {[styles.sticky]: isSticky})}
```